bitnami/nginx/templates/servicemonitor.yaml

```
@@ -22,6 +29,14 @@ spec:
       {{- if .Values.metrics.serviceMonitor.scrapeTimeout }}
       scrapeTimeout: {{ .Values.metrics.serviceMonitor.scrapeTimeout }}
       {{- end }}
+      {{- if $.Values.metrics.podMonitor.relabelings }}
+      relabelings:
+      {{- include "common.tplvalues.render" (dict "value" .Values.metrics.podMonitor.relabelings "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if $.Values.metrics.podMonitor.relabelings }}
+      metricRelabelings:
+      {{- include "common.tplvalues.render" (dict "value" .Values.metrics.podMonitor.metricRelabelings "context" $) | nindent 8 }}
+      {{- end }}      
```

The `$.` isn't required:
```suggestion
      {{- if .Values.metrics.podMonitor.relabelings }}
      relabelings:
      {{- include "common.tplvalues.render" (dict "value" .Values.metrics.podMonitor.relabelings "context" $) | nindent 8 }}
      {{- end }}
      {{- if .Values.metrics.podMonitor.relabelings }}
      metricRelabelings:
      {{- include "common.tplvalues.render" (dict "value" .Values.metrics.podMonitor.metricRelabelings "context" $) | nindent 8 }}
      {{- end }} 
```